src/resmom/mom_main.c

```diff
@@ -3067,17 +3066,12 @@ do_mom_action_script(int	ae,	/* index into action table */
 		sprintf(buf, "%d", ptask->ti_qs.ti_sid);
 		bld_env_variables(&vtable, "PBS_SID", buf);
 		/* PBS_JOBDIR */
-		if ((pjob->ji_wattr[(int)JOB_ATR_sandbox].at_flags &
-			ATR_VFLAG_SET) &&
-			(strcasecmp(pjob->ji_wattr[JOB_ATR_sandbox].
-			at_val.at_str, "PRIVATE") == 0)) {
+		if (is_jattr_set(pjob, JOB_ATR_sandbox)
```

Comment:
Here as well.

Reply:
this would be too long a line (119 chars), so I'll leave this one as it is.